rpc/serializers.py

```diff
@@ -378,30 +378,61 @@ def get_slug(self, obj) -> str:
 
 @dataclass
 class SubseriesDoc:
-    type: str
-    number: int
-    members: list[int]
+    name: str
+    rfc_number: int
 
-    @property
-    def rfc_count(self):
-        return len(self.members)
 
-    @property
-    def display_name(self):
-        return f"{self.type.upper()} {self.number}"
+class SubseriesDocSerializer(serializers.Serializer):
+    name = serializers.CharField()
+    rfc_number = serializers.IntegerField()
 
-    @property
-    def slug(self):
-        return f"{self.type.lower()}{self.number}"
 
+@dataclass
+class Subseries:
+    type: str
+    number: int
 
-class SubseriesDocSerializer(serializers.Serializer):
+
+class SubseriesSerializer(serializers.Serializer):
     type = serializers.CharField()
     number = serializers.IntegerField()
-    members = serializers.ListField(child=serializers.IntegerField())
-    rfc_count = serializers.IntegerField(read_only=True)
-    slug = serializers.CharField(read_only=True)
-    display_name = serializers.CharField(read_only=True)
+    documents = serializers.SerializerMethodField()
+    rfc_count = serializers.SerializerMethodField()
+    slug = serializers.SerializerMethodField()
+    display_name = serializers.SerializerMethodField()
```

Comment:
The `SerializerMethodField` methods really ought to be properties on the `Subseries` dataclass (whatever it is called) rather than in the serializer. This will make it easier to use the nested serializers.